src/component/mxgraph/shape/activity-shapes.ts

```diff
@@ -242,27 +242,43 @@ export class CallActivityShape extends BaseActivityShape {
     super(bounds, fill, stroke, strokewidth);
   }
 
-  override paintBackground(c: mxAbstractCanvas2D, x: number, y: number, w: number, h: number): void {
+  override paintForeground(c: mxAbstractCanvas2D, x: number, y: number, w: number, h: number): void {
+    super.paintForeground(c, x, y, w, h);
+
+    const paintParameter = buildPaintParameter({ canvas: c, x, y, width: w, height: h, shape: this });
```

Comment:
ℹ️ we may move the following code to a `paintIcon` function to make the intent more explicit

Reply:
I wanted to do that but the `paintIcon` function is in `TaskShape` and the **Global Tasks** are not considered as **Task** on the BPMN specification.
We could move the `paintIcon` function is in `BaseTaskShape`, but I don't know the impacts.

Reply:
ok, let's keep the code as it is now, we can refactor later.